src/stories/Library/slider/Slider.tsx

```diff
@@ -0,0 +1,75 @@
+import { FC, ReactNode, useEffect } from "react";
+import { Helmet } from "react-helmet";
+
+type SliderProps = {
+  title?: string;
+  items: ReactNode[];
+};
+
+const Slider: FC<SliderProps> = ({ title, items }) => {
```

Comment:
Just commenting it here. 

This could be storybook specific, havenøt testet in drupal yet. 

The slider seems to only be browsable with clicking on the arrow, i guess it should be scrollable too for better ux?

Reply:
This is something I'd like to talk with DDF about during a demo - for now, i'll keep it like this.